flow-libs/browserslist.js.flow

```diff
@@ -0,0 +1,13 @@
+// @flow strict-local
+
+declare module 'browserslist' {
+  import type {ConfigResult} from '@parcel/types';
+
+  declare type BrowserslistConfig = {|[string]: string | Array<string>|};
+
+  declare module.exports: {|
+    (browsers: string | Array<string>): Array<string>,
+    parseConfig(configResult: ConfigResult): Promise<?BrowserslistConfig>,
+    loadConfig({[path: FilePath]: FilePath, ...}): string,
```

Comment:
not sure how `FilePath` works here. A string is probably better, or we could define `FilePath` inside this module as an alias to `string`.